include/ck/utility/data_type.hpp

```diff
@@ -996,6 +1411,18 @@ inline __host__ __device__ constexpr bhalf_t type_convert<bhalf_t, float>(float
     return uint16_t(u.int32 >> 16);
 }
 
+template <>
```

Comment:
Do we really need type_convert between `int4_t` and `int8_t` ?

Reply:
Ultimately we will probably have some kind of quantization between those two types. @zjing14 has more details.